applications/CoSimulationApplication/python_scripts/coupling_interface_data.py

```diff
@@ -64,15 +70,9 @@ def Initialize(self):
                 if domain_size != self.dimension:
                     cs_tools.cs_print_warning('CouplingInterfaceData', '"DOMAIN_SIZE" ({}) of ModelPart "{}" does not match dimension ({})'.format(domain_size, self.GetModelPart().Name, self.dimension))
 
-        # location of data on ModelPart
-        self.location = self.settings["location"].GetString()
-        admissible_locations = ["node_historical", "node_non_historical","element","condition","process_info","model_part"]
-        if not self.location in admissible_locations:
-            raise Exception('"{}" is not allowed as "location", only the following options are possible:\n{}'.format(self.location, ", ".join(admissible_locations)))
-
         if self.location == "node_historical":
             if not self.GetModelPart().HasNodalSolutionStepVariable(self.variable):
-                raise Exception('"{}" is missing as SolutionStepVariable in ModelPart "{}"'.format(variable_name, self.GetModelPart().Name))
+                raise Exception('"{}" is missing as SolutionStepVariable in ModelPart "{}"'.format(self.variable.Name(), self.GetModelPart().Name))
```

Comment:
will this `self.variable.Name()` work in python only version  ?

Reply:
yes => https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CoSimulationApplication/custom_data_structure/pyKratos/Variables.py#L12